### PR TITLE
APIO-109 Validate representable name

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by ahdezma** [comment]
ci:test

File: apio-architect-impl/src/test/java/com/liferay/apio/architect/impl/wiring/validator/NameValidatorImpTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+import com.liferay.apio.architect.impl.wiring.osgi.validator.NameValidator;
+import com.liferay.apio.architect.impl.wiring.osgi.validator.NameValidatorImpl;
+
+import org.junit.Test;
+
+/**
+ * @author Víctor Galán
+ */
+public class NameValidatorImpTest {
```
Comment (ahdezma) [inline apio-architect-impl/src/test/java/com/liferay/apio/architect/impl/wiring/validator/NameValidatorImpTest.java]: `nameValidator` could be an static class field for all tests. Then you could create a `validate` method like:

```java
private static boolean _validate(String name) {
    return _nameValidator.validate(name)
}
```

and your tests could be smaller:

```java
assertThat(validate(null), is(false));
assertThat(validate("123blog"), is(false));
```

Also I think we should have a test for testing that the validator message is correct.

**Comment by victorg1991** [comment]
Suggestions applied :D

**Comment by victorg1991** [comment]
Thanks!

**Pull request merged**